package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "ils-app",
+  "version": "0.1.0",
+  "private": true,
```

Comment:
why this?

Reply:
copied from ILS, I guess someone wanted to prevent from publishing?